README.md

```diff
@@ -1,27 +1,43 @@
-# cidr-filtering-cni
+# bridge-filtering
 
-This plugin allows user to define allow list for ingress and egress packet filtering for a Kubernetes pod.
+This CNI plugin allows the user to control traffic flow at the IP address or port level (OSI layer 3 or 4) for particular applications in the Kubernetes cluster, thus specifying how a pod is allowed to communicate with various network "entities". The pod interface must be a port of a bridge (for example a kubevirt virtual machine connected with bridge binding method). By default, all ingress and egress traffic is denied.
+Users may create special `ConfigMaps` in the pod namespace to indicate allowed ingress or egress connections.
+This can be done for network layers L3 and L4. The supported L4 protocols are UDP and TCP.
+
+Since the nftable rules implementing traffic filtering are created when a pod is being created, the CNI cannot update the provisioned rules if those are updated in the `ConfigMaps`.
```

Comment:
```suggestion
Since the nftable rules implementing traffic filtering are only applied when a pod is created, the CNI **cannot** update the provisioned rules if those are updated in the `ConfigMaps`.
```

I personally usually like to highlight words like cannot / will not etc. Makes it harder to miss.

Reply:
[Thanks, Done](https://github.com/rhrazdil/cidr-filtering-cni/pull/13/commits/9fb106b692fb73da4518627a34020acbfc01a52b)